packit/cli/propose_downstream.py

```diff
@@ -11,20 +11,38 @@
 import click
 
 from packit.cli.types import LocalProjectParameter
-from packit.cli.utils import cover_packit_exception, iterate_packages, get_packit_api
+from packit.cli.utils import (
+    cover_packit_exception,
+    iterate_packages,
+    get_packit_api,
+)
 from packit.config import pass_config, get_context_settings, PackageConfig
 from packit.config.aliases import get_branches
 from packit.constants import (
     PACKAGE_LONG_OPTION,
     PACKAGE_SHORT_OPTION,
     PACKAGE_OPTION_HELP,
+    CENTOS_STREAM_DISTGIT_URL,
 )
+from packit.config.job_config import JobType
 
 logger = logging.getLogger(__name__)
 
 
 def get_dg_branches(api, dist_git_branch):
     cmdline_dg_branches = dist_git_branch.split(",") if dist_git_branch else []
+
+    # TODO(Gris): Are we support mixing c9s with fedora-all ?
```

Comment:
I think it's fine to mix those, but:

* As I see, this will require improvements in how we work with `dist_git_base_url`.
* We can leave this to a follow-up.

Reply:
V2 patch support placing `c9s` with `fedora-all`.